package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gooddata/react-components",
-  "version": "7.5.0-beta.49",
+  "version": "7.5.0",
```

Comment:
Please leave the version unchanged. This will be updated by the build server appropriately.